tests/initializers/linear/sinkhorn_init_test.py

```diff
@@ -99,12 +99,16 @@ def run_sinkhorn(
     init = linear_init.SortingInitializer(**kwargs)
   elif initializer == "gaussian":
     init = linear_init.GaussianInitializer(**kwargs)
+  elif initializer == "subsample":
+    init = linear_init.SubsampleInitializer(**kwargs)
   else:
     raise NotImplementedError(initializer)
 
   geom = pointcloud.PointCloud(x, y, epsilon=epsilon)
   prob = linear_problem.LinearProblem(geom, a, b)
-  solver = sinkhorn.Sinkhorn(lse_mode=lse_mode, initializer=init)
+  # can jit initializer + solver or just solver, but not both
+  # setting solver to not jit, and testing just jitting everyting
+  solver = sinkhorn.Sinkhorn(lse_mode=lse_mode, initializer=init, jit=False)
```

Comment:
I think the comment + `jit=False` can be removed after #310 .
I will look into more detail for the jitting issues in #312 (and possibly remove it from the solvers, again).